### PR TITLE
feat: Add Hugo link

### LINKED ISSUE #1709: [Add] Hugo
### Site URL

https://gohugo.io/

### Description of the above provided site

If you're looking for a framework to help you build your documentation site quickly, check out Hugo.  It allows you to output your content in multiple formats, including JSON or AMP as well as provides shortcodes to simplify your Markdown files. 

### Category

Other

### Sub Category

Tools 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by R055A**
Closes #1709 

## Changes proposed

Add link for Hugo framework website to the 'Other' category under a new subcategory named 'Other Tools' (URL: other/other-tools)

## Screenshots

one

two

## Note to reviewers

Hacktoberfest

### CODE REVIEW

File: database/other/other-tools.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "name": "Hugo",
+    "description": "Hugo is a popular open-source static site generator that offers blistering speed, robust content management, shortcodes, built-in templates, multilingual and i18n support, and more. Learn how to get started, see the showcase, and install Hugo in seconds on any platform or server",
```
Comment (CBID2) [inline database/other/other-tools.json:4]:
```suggestion
    "description": "If you're looking for a framework to help you build your documentation site quickly, check out Hugo. It allows you to output your content in multiple formats, including JSON or AMP as well as provides shortcodes to simplify your Markdown files.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @R055A! :) Just made some adjustments to the description. Other than that, you're off to a great start! :)

**Review by CBID2 — DISMISSED** [review]
Well done @R055A! :) @Anmol-Baranwal, it's your turn! :)

**Review by Anmol-Baranwal — COMMENTED** [review]
@CBID2 
Rather than creating a new subcategory, can't we add this in `Dev Tools` subcategory in 

image

What do you think?

**Comment by CBID2** [comment]
> @CBID2 
> 
> Rather than creating a new subcategory, can't we add this in `Dev Tools` subcategory in 
> 
> 
> 
> image
> 
> 
> 
> What do you think?

I don't know @Anmol-Baranwal. My intention with technical writing subcategory is to create a hub of tools where people who have that role can find what they need, especially since they are not often spotlighted in conversations about roles to pursue in tech.

**Comment by Anmol-Baranwal** [comment]
> I don't know @Anmol-Baranwal. My intention with technical writing subcategory is to create a hub of tools where people who have that role can find what they need, especially since they are not often spotlighted in conversations about roles to pursue in tech.

See the image below.
image


It is in `Other/other tools` not `technical writing`

**Comment by CBID2** [comment]
> > I don't know @Anmol-Baranwal. My intention with technical writing subcategory is to create a hub of tools where people who have that role can find what they need, especially since they are not often spotlighted in conversations about roles to pursue in tech.
> 
> 
> 
> See the image below.
> 
> image
> 
> 
> 
> 
> 
> It is in `Other/other tools` not `technical writing`

Thanks for pointing that out @Anmol-Baranwal! :) I still want it in `Technical Writing/Tools` though.

**Comment by CBID2** [comment]
Yeah @R055A. When I raised this issue, that option was not in the issue for it. There's a PR that's going to address it soon. For now, just add it to that category.

**Comment by CBID2** [comment]
> technical-writing/tools

technical-writing/tools @R055A

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
@R055A 
Change the category of `Hugo` to `Technical Writing/Technical Writing Tools` and remove this subcategory `Other Tools` as mentioned in the image below.

Remove it from here:

image


and add it here:

image

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me. 

> @R055A thanks for the changes.

**Review by CBID2 — APPROVED** [review]
Well done @R055A

**Pull request merged**